_dashboards/discover/exploring-query-enhancements.md

```diff
@@ -0,0 +1,77 @@
+
+You can leave your feedback at [https://forum.opensearch.org/)](https://forum.opensearch.org/) to help the OpenSearch open source project improve this feature.
+{: .note}
+
+## Enabling query enhancements
+
+To enable the query enhancements through OpenSearch Dashboards, follow these steps: 
+
+1. Go to **Dashboards Management** > **Advanced settings** > **Search** and toggle on **Enable query enhancements**. Tip: You can select the **Search** pane from the **Category** dropdown menu in the upper-right search bar.
+2. Select the **Save** button to save your changes. 
+3. Reload the page as prompted in the pop-up message.
+
+Alternatively, you can override the setting on startup by running the following command:
+
+```
+./bin/opensearch-dashboards --uiSettings.overrides['query:enhancements:enabled']=true
+```
+{% include copy-curl.html %}
+
+## Using the experimental features
+
+The following tutorials guide you through some of the experimental features and capabilities.
+
+### Query language enhancements
+
+You can now use PPL in **Discover**. Follow these steps to try out the feature:
+
+1. Go to **Discover** and select **PPL** from the query language dropdown menu in the upper-right search bar. You should see a dashboard containing the query editor, histogram, and data table panes.
+2. Select a sample dataset. For this example, select `opensearch_dashboards_sample_data_ecommerce` from the data source dropdown menu above the query editor and adjust the time filter to **Last 1 year**.
+3. Enter the following example PPL query:
+
+```json
+source = opensearch_dashboards_sample_data_logs
+| where tags = "success" 
+| where geo.dest = "US"
+```
+{% include copy-curl.html %}
+
+4. View the resulting output that shows the number of successful log entries originating from the United States. You should see an updated histogram and data table following the query editor. 
+5. Select the **Recent queries** option within the query editor toolbar to display your recent queries.
+
+PPL and DQL provide an autocomplete option that suggests field names, functions, and syntax. 
+
+
+## Selecting data sources and data types through the UI
+
+You can now select your data sources and types from within the **Discover** dashboard. Follow these steps to try out the feature.
+
+1. From the **Discover** page, select a data source from the dropdown menu in the upper toolbar. 
+2. Select the **View all available data** button to display a list of your available data sources. You may need to refresh your page to display any newly added data sources.
+3. Select the desired data source and follow steps displayed in the data sources window to manage your data source.
+
+You can now use **Query Assist** with PPL queries. With **Query Assist**, you can ask questions like _Are there any errors in my logs?_. The assistant includes predefined prompts. Follow these steps to try out the feature:
```

Comment:
didn't see it but should we call out that Query Assist for PPL but not necessarily available for all data sources

Reply:
@kavilla Should we tell user which specific data sources/types it's available for?

Reply:
Query Assist is only available in query enhancements when the data source itself supports it. It's dependent on if the user adds a cluster with query assist enabled/set up. I think we can call out that query assist is required on the OpenSearch data source, something like query assist is available when included data source has it available?